src/datanode/src/datanode.rs

```diff
@@ -47,6 +47,35 @@ impl Default for ObjectStoreConfig {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct WalConfig {
+    // wal directory
+    pub dir: String,
+    // wal file size in bytes
+    pub file_size: usize,
+    // wal purge threshold in bytes
+    pub purge_threshold: usize,
+    // purge interval in seconds
+    pub purge_interval: u64,
```

Comment:
We have imported the `humantime-serde` crate, we could use it.
https://github.com/GreptimeTeam/greptimedb/blob/72f05a3137bdedf0e7502a246a2533ded8d104cc/src/servers/src/http.rs#L111-L112